harmonica/tests/test_tesseroid.py

```diff
@@ -159,6 +159,25 @@ def test_invalid_tesseroids():
             _check_tesseroids(new_tesseroids)
 
 
+def test_disable_checks():
+    "Check if the disable_checks flag works properly"
+    valid_tesseroid = [0, 10, 0, 10, 10, 20]
+    invalid_tesseroid = [0, 10, 0, 10, 20, 10]
+    density = 100
+    coordinates = [0, 0, 10]
+    # Check if an invalid tesseroid doesn't raise an error with the disable_checks flag set to True
+    tesseroid_gravity(
+        coordinates, invalid_tesseroid, density, field="potential", disable_checks=True
+    )
+    # Check if an invalid tesseroid returns the inverse of a valid one's return value
+    # with only top and bottom values flipped between the two.
+    assert tesseroid_gravity(
+        coordinates, invalid_tesseroid, density, field="potential", disable_checks=True
+    ) == (-1) * tesseroid_gravity(
+        coordinates, valid_tesseroid, density, field="potential"
+    )
```

Comment:
I would change it in a similar way I suggested for the prism case: store the resulting potential generated by passing the invalid tesseroid.